esp/src/eclwatch/UserQueryWidget.js

```diff
@@ -643,6 +797,71 @@ define([
             }
         },
 
+        ensurePermissionsPane: function (id, params) {
+            id = this.createChildTabID(id);
+            var retVal = registry.byId(id);
+            if (!retVal) {
+                retVal = new ShowIndividualPermissionsWidget({
+                    id: id,
+                    title: params.Name,
```

Comment:
I don't pass in TabName in regular ensurePermissionsPane() just in the ones I don't have a scope in.